### DISCUSSION TITLE
How to add your own actor

### DISCUSSION THREAD

**Discussion by chenzhoule-6** [microsoft/AirSim#4523] [Support Q&A]
Can you add your own actor through airsim source code, except for the default car and UAV,and you can specify the location of the actor

**Comment by zimmy87** [comment]
Yes, [this tutorial](https://www.youtube.com/watch?v=Bp86WiLUC80) shows how to add a custom vehicle blueprint with your own mesh. You can modify this vehicle blueprint however you like, however to work with either `Car` or `Multirotor` SimModes, the blueprint will have to extend either CarPawn or FlyingPawn. To use a different base actor type, you will need to either modify the existing SimModes or create a new one.
